module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -397,7 +397,14 @@ public function getStatuses($ids)
      */
     public function getHolding($id, array $patron = null, array $options = [])
     {
-        return $this->getItemStatusesForBiblio($id, $patron);
+        $biblio = $this->getBiblio($id);
+        $type   = $biblio['item_type'];
+        if (isset($this->config['Holdings']['itemLimit'][$type])) {
```

Comment:
It's probably a good idea to initialize $options... and you could use null coalescing to avoid the if/else. I'd change this whole conditional block to something like:

```
$options = [
    'itemLimit' => $this->config['Holdings']['itemLimit'][$type]
        ?? $this->config['Holdings']['itemLimit']['DEFAULT']
        ?? ''
];
```

(Note that this also incorporates my suggestion above about a default fallback).

Reply:
thanks @demiankatz I've added that in my follow-up!